RadixWallet/Clients/OnLedgerEntitiesClient/OnLedgerEntitiesClient+Interface.swift

```diff
@@ -477,4 +477,27 @@ extension OnLedgerEntitiesClient {
 			nonXrdResources: nonXrdResourceDetails
 		)
 	}
+
+	public func isStakeClaimNFT(_ resource: OnLedgerEntity.Resource) async -> OnLedgerEntity.Validator? {
```

Comment:
I like that we only have one method, but then the name should probably be "get-", although Apple explicitly recommends against using "get" for async methods. I think it could just be `stakeClaimNFT`